Description: Do not build with -Werror.
 It is a time bomb. And fails with Debian's CFLAGS.
Origin: vendor
Bug: https://sourceforge.net/p/autogen/bugs/185/
Last-Update: 2019-09-21

--- configure.ac
+++ configure.ac
@@ -198,7 +198,7 @@ fi
 WARN_CFLAGS=
 test "X${GCC}" = Xyes && {
   CFLAGS="$CFLAGS -Wno-format-contains-nul -fno-strict-aliasing"
-  WARN_CFLAGS="$CFLAGS "`echo -Wall -Werror -Wcast-align -Wmissing-prototypes \
+  WARN_CFLAGS="$CFLAGS "`echo -Wall -Wcast-align -Wmissing-prototypes \
 	-Wpointer-arith -Wshadow -Wstrict-prototypes -Wwrite-strings \
 	-Wstrict-aliasing=3 -Wextra -Wno-cast-qual`
 }]dnl -Wconversion -Wsign-conversion -Wstrict-overflow
